### DISCUSSION TITLE
Web Accesibility question on Tui ComboBox and Select

### DISCUSSION THREAD

**Discussion by Joshua-Ryan** [taiga-family/taiga-ui#6701] [Q&A] [answered]
According to the [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/), for single select combo boxes, the tab button is supposed to select the currently active descendant element and then move to the next element in the DOM. 

In the current release of taiga v3.65.0 I noticed that space bar and enter work to select items, but tab does not. I am wondering if there is a way to determine the active descendant (without querying the DOM) in a tui-data-list-wrapper or combo box so that I may patch that data after finding the select option it correlates to.

**Comment by waterplea** [comment, marked as answer]
Taiga UI has active descendant actually focused at all time — instead of having the select component focused and active descendant appear focused. So you can listen to keydown.tab event. But you would only get the DOM element, not the underlying tuiOption. Better to create a feature request. However I don't think it's a good idea and I haven't seen a UI library that does that. Native Select element just always has active descendent selected, it does not allow you to navigate around options with arrows and then decide not to change value. So I would say it's a mistake in W3C part.
